src/applications/vaos/services/appointment/index.js

```diff
@@ -264,6 +264,19 @@ export function getPatientTelecom(appointment, system) {
  * @returns {Boolean} Whether or not the facility has a COVID vaccine phone line
  */
 export function hasValidCovidPhoneNumber(facility) {
```

Comment:
`hasValidCovidPhoneNumber` is also being used in the `ContactFacilitiesPage` component to determine the number to display for `FacilityPhone`. We should be using your new `hasTransformedCovidPhoneNumber` method there.

Reply:
I think we should go a step further than that, even. Functions in services/appointment/index.js are typically working with transformed data. They're not usually checking things in the format we get from the server.

I would say that the new `hasTransformedCovidPhoneNumber` should be called `hasValidCovidPhoneNumber`, and the check in `hasValidCovidPhoneNumber` should be moved into the transformer file. And then any component code should use the new `hasValidCovidPhoneNumber`.

Reply:
@jbalboni, Makes a lot of sense moving the check over to the transformer file. Thanks for explaining that functions in `services/appointment/index.js` are working with transformed data.